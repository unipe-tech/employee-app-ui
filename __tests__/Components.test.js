import React from "react";
import renderer from "react-test-renderer";
import PrimaryButton from "../components/PrimaryButton";
import DateEntry from "../components/DateEntry";

describe("PrimaryButton", () => {
  test("if button renders correctly", () => {
    const tree = renderer.create(<PrimaryButton />).toJSON;
    expect(tree).toMatchSnapshot();
  });
});

describe("DateEntry", () => {
  test("if DateEntry renders correctly", () => {
    const tree = renderer.create(<DateEntry />).toJSON;
    expect(tree).toMatchSnapshot();
  });
});
