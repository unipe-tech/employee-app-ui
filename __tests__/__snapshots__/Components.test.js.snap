// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PrimaryButton if button renders correctly 1`] = `[Function]`;
